self_supervised_pretraining/ssl_finetune.ipynb

```diff
@@ -5,23 +5,50 @@
    "id": "14fc43c2",
    "metadata": {},
    "source": [
+    "# Finetune 3D Multi-Class Abdominal Segmentation Model Using SSL Pre-trained Weights\n",
```

Comment:
The license header is usually placed at the top. @finalelement may I ask you to put the title `# Finetune 3D Multi-Class Abdominal Segmentation Model Using SSL Pre-trained Weights` below the license header as the [example](https://github.com/Project-MONAI/tutorials/blob/main/.github/contributing_templates/notebook/example_feature.ipynb)? Thanks!

Reply:
Good catch, addressed.